voluseg/_steps/step1.py

```diff
@@ -56,7 +61,14 @@ def make_output_volume(name_volume, volume):
                 # fix dimensionality
                 if volume.ndim == 2:
                     volume = volume[None, :, :]
-                volume = volume.transpose(2, 1, 0)
+
+                # reorder dimensions to xyz, if necessary
+                if p.dim_order == "zyx":
```

Comment:
Minor comment. At some stage we should add all possible orders here:
* xyz
* xzy
* yxz
* yzx
* zxy
* zyx